### ISSUE TITLE
Pytradfri not working in version 1.9 of gateway?

### ISSUE THREAD

**Issue by ggravlingen** [ggravlingen/pytradfri#246]
Anecdotal evidence in Home Assistant's forums and Discord channels suggests pytradfri is not working in version 1.9 of the gateway.

```
File "/home/homeassistant/homeassistant/components/http/real_ip.py", line 40, in real_ip_middleware
    return await handler(request)
  File "/home/homeassistant/homeassistant/components/http/ban.py", line 73, in ban_middleware
    return await handler(request)
  File "/home/homeassistant/homeassistant/components/http/auth.py", line 231, in auth_middleware
    return await handler(request)
  File "/home/homeassistant/homeassistant/components/http/view.py", line 128, in handle
    result = await result
  File "/home/homeassistant/homeassistant/components/config/config_entries.py", line 164, in post
    return await super().post(request, flow_id)
  File "/home/homeassistant/homeassistant/components/http/data_validator.py", line 48, in wrapper
    result = await method(view, request, *args, **kwargs)
  File "/home/homeassistant/homeassistant/helpers/data_entry_flow.py", line 90, in post
    result = await self._flow_mgr.async_configure(flow_id, data)
  File "/home/homeassistant/homeassistant/data_entry_flow.py", line 89, in async_configure
    result = await self._async_handle_step(flow, cur_step["step_id"], user_input)
  File "/home/homeassistant/homeassistant/data_entry_flow.py", line 132, in _async_handle_step
    result: Dict = await getattr(flow, method)(user_input)
  File "/home/homeassistant/homeassistant/components/tradfri/config_flow.py", line 55, in async_step_auth
    self.hass, host, user_input[KEY_SECURITY_CODE]
  File "/home/homeassistant/homeassistant/components/tradfri/config_flow.py", line 164, in authenticate
    key = await api_factory.generate_psk(security_code)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/pytradfri/api/aiocoap_api.py", line 195, in generate_psk
    self._psk = await self.request(command)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/pytradfri/api/aiocoap_api.py", line 155, in request
    result = await self._execute(api_commands)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/pytradfri/api/aiocoap_api.py", line 146, in _execute
    _, res = await self._get_response(msg)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/pytradfri/api/aiocoap_api.py", line 95, in _get_response
    r = await pr.response
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/protocol.py", line 720, in _run_outer
    await cls._run(app_request, response, weak_observation, protocol, log)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/protocol.py", line 745, in _run
    await protocol.find_remote_and_interface(app_request)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/protocol.py", line 295, in find_remote_and_interface
    if await ri.fill_or_recognize_remote(message):
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/tokenmanager.py", line 179, in fill_or_recognize_remote
    return await self.token_interface.fill_or_recognize_remote(message)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/messagemanager.py", line 327, in fill_or_recognize_remote
    remote = await self.message_interface.determine_remote(message)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/transports/tinydtls.py", line 328, in determine_remote
    dtlsparams = self.ctx.client_credentials.credentials_from_request(request)
  File "/home/pi/.homeassistant/deps/lib/python3.7/site-packages/aiocoap/credentials.py", line 285, in credentials_from_request
    raise CredentialsMissingError("No suitable credentials for %s" % uri)
aiocoap.credentials.CredentialsMissingError: No suitable credentials for coaps://10.11.12.30:5684/15011/9063
2019-09-15 19:20:44 ERROR (MainThread) [coap] Connection loss was not expected.
```

**Comment by balloob** [comment]
Try upgrading the dependencies aiocoap and install the extras: `aiocoap[tinydtls]`

**Comment by ggravlingen** [comment]
This has been resolved in pytradfri 6.3.1. The error was due to incorrect handling of dependencies in pytradfri.
